docs/docker.rst

```diff
@@ -22,65 +18,128 @@ Usage
-         --device=/dev/kfd --device=/dev/dri                      \
-         --security-opt seccomp=unconfined --group-add video      \
-         -v $(pwd)/results:/milabench/envs/runs                   \
-         ghcr.io/mila-iqia/milabench:rocm-nightly                 \
-         milabench summary /milabench/envs/runs
+For the performance report, it is the same command:
+
+.. code-block:: bash
+
+   # Show Performance Report
+   docker run -it --rm                             \
+         -v $(pwd)/results:/milabench/envs/runs    \
+         $MILABENCH_IMAGE                          \
+         milabench report --runs /milabench/envs/runs
+
+
+Multi-node benchmark
+^^^^^^^^^^^^^^^^^^^^
+
+There is currently one multi-node benchmark, ``opt-2_7b-multinode``. Here is how to run it:
+
+* Provided you have the following two machines that can see each other on the network:
+  * ``manager-node``
+  * ``worker-node``
+* ``docker pull`` the image on both nodes.
+* Prior to running the benchmark, create a SSH key pair on ``manager-node`` and set up public key authentication to ``worker-node``.
+* On ``manager-node``, execute ``milabench run`` via Docker.
+  * Mount the private key at ``/milabench/id_milabench`` in the container
+  * Use ``--override`` statements as shown below to tell milabench about both nodes
+
+The command should look something like this:
 
+.. code-block:: bash
+
+    # On manager-node:
+
+    # Modify these variables to match your setup
+    export SSH_KEY=$HOME/.ssh/id_rsa
+    export NODE1=manager-node
+    export NODE2=worker-node
+    export NUM_MACHINES=2
+    export MILABENCH_USER=$USER  # The user on worker-node that public key auth is set up for
+
+    docker run -it --rm --gpus all --network host --shm-size 32G \
+      -v $SSH_KEY:/milabench/id_milabench \
+      -v $(pwd)/results:/milabench/envs/runs \
+      $MILABENCH_IMAGE \
+      milabench run \
+      --override opt-2_7b-multinode.docker_image='"'$MILABENCH_IMAGE'"' \
+      --override opt-2_7b-multinode.manager_addr='"'$NODE1'"' \
+      --override opt-2_7b-multinode.worker_addrs='["'$NODE2'"]' \
+      --override opt-2_7b-multinode.worker_user='"'$MILABENCH_USER'"' \
+      --override opt-2_7b-multinode.num_machines='"'$NUM_MACHINES'"' \
+      --capabilities nodes=$NUM_MACHINES \
+      --select opt-2_7b-multinode
 
-Publish
--------
+The last line (``--select opt-2_7b-multinode``) specifically selects the multi-node benchmark. Omit that line to run all benchmarks.
+
+For 4 nodes, use ``--override opt-2_7b-multinode.worker_addrs='["'$NODE2'","'$NODE3'","'$NODE4'"]'`` (and of course ``NUM_MACHINES=4``).
+
+
+Building images
+---------------
 
 Images can be build locally for prototyping and testing.
 
 .. code-block::
 
-   sudo docker build -t milabench:cuda-nightly --build-arg ARCH=cuda --build-arg CONFIG=ci.yaml .
+   sudo docker build -t milabench:cuda-nightly --build-arg ARCH=cuda --build-arg CONFIG=standard.yaml .
```

Comment:
I'm pretty sure that doesn't work since the Dockerfiles are under `docker/` and are not named `Dockerfile`.  This would need an additional `-f docker/Dockerfile-{cuda,rocm}` and dropping the ARCH build-arg

Reply:
Oh true. I think that was changed after that part of the doc had been written.